spec/support/factory_girl.rb

```diff
@@ -7,7 +7,7 @@
     password_confirmation "password"
   end
 
-  factory :food_truck do
+  factory :food_truck, :class => FoodTruck do |f|
```

Comment:
Unused block argument - `f`. You can omit the argument if you don't care about it.<br>Use the new Ruby 1.9 hash syntax.